.lando.yml

```diff
@@ -140,6 +140,8 @@ services:
         DRUPAL_CLIENT_VIEWER_ID: drupal-client-viewer-id
         DRUPAL_CLIENT_VIEWER_SECRET: drupal_client_viewer_secret_not_secure_used_only_locally
         NEXT_PUBLIC_DRUPAL_BASE_URL: https://next-drupal-starterkit.lndo.site
+        DRUPAL_BASE_URL_INTERNAL: http://appserver_nginx.nextdrupalstarterkit.internal
+        DRUPAL_BASE_URL_INTERNAL_IMAGES: http://appserver_nginx.nextdrupalstarterkit.internal
```

Comment:
In lando also just `http://appserver_nginx` should work, I don't think the `.nextdrupalstarterkit.internal` part is needed. I suppose it's one less thing that would need to be changed later

Reply:
Done in 339c79a